src/assets/translations/fr/main.json

```diff
@@ -657,7 +666,8 @@
         "stake": "Miser",
         "withdraw": "Retirer",
         "unstakeRequest": "Demande de démise",
-        "setRuneAddress": "Définir l'adresse Rune"
+        "setRuneAddress": "Définir l'adresse Rune",
```

Comment:
That's wrong in the EN flavor of this copy but should be `RUNE` if you feel like updating it while at it 🙏🏽

Reply:
Corrected in all languages including English.